wdio.conf.ts

```diff
@@ -1,5 +1,21 @@
 import { uploadExtractDataFileForUITest } from "./ui-tests/testData/test.setup";
 
+type Environment = "dev" | "build" | "staging";
+
+const baseUrls = {
+  local: "http://localhost:3000",
+  dev: "https://btm.dev.account.gov.uk/",
+  build: "https://btm.build.account.gov.uk/",
+  staging: "https://btm.staging.account.gov.uk/",
+};
+
+const env =
+  (process.env.ENV_NAME as Environment) in baseUrls
+    ? (process.env.ENV_NAME as Environment)
+    : "local";
+
+const baseUrl = baseUrls[env];
```

Comment:
What's the advantage over doing:

```suggestion
const determineBaseUrl = ():string => {
  switch (process.env.ENV_NAME) {
    case "dev":
      return "https://btm.dev.account.gov.uk/"
    case "build":
      return "https://btm.build.account.gov.uk/"
    case "staging":
      return "https://btm.staging.account.gov.uk/"
    default:
      return "http://localhost:3000";
  }
}

const baseUrl = determineBaseUrl();
```

Seems more straight forward over a typecast-lookup? Or do you need the type and baseUrls for anything else?

Reply:
no, not using type and base url in any other place, changed to simplified version